desc/profiles.py

```diff
@@ -768,6 +766,9 @@ def from_values(
             profile in power series basis fit to given data.
 
         """
+        if sym:
+            x = x**2
+            order = order // 2
```

Comment:
I don't think we want to reduce the order here without padding the params later, or passing in the mode numbers explicitly